src/main/java/edu/cornell/kfs/fp/service/RecurringDisbursementVoucherDocumentService.java

```diff
@@ -50,4 +52,34 @@
 	 */
 	boolean autoApproveDisbursementVouchersSpawnedByRecurringDvs();
 
+    /**
+     * Cancels all DVs spawned by the Recurring Disbursement Voucher that are in saved status.
+     * @param recurringDisbursementVoucherDocument
+     * @param cancelMessage
+     * @return A Set<String> of DV document numbers that have been canceled.
+     */
+    Set<String> cancelDisbursementVouchers(RecurringDisbursementVoucherDocument recurringDisbursementVoucherDocument, String cancelMessage);
```

Comment:
this method name should be updated to reflect the fact that it only cancels saved DVs. Updating the name will also make the code where this is called easier to read and understand